### PR TITLE
Add validation on "to" interval time to be greater than "from"

### LINKED ISSUE #270: Missing intervals validation
1. offer transport
2. select fixed intervals
3. select from 10:30 PM to 10:00 PM
	
**Expected**: invalid time range
**Actual**: field remain valid?

### PR DESCRIPTION
**Pull request by costinicolescu**
### What does it fix?

Closes #270  

Validation for `availability_interval_to` was added to be greater than `availability_interval_from`.
New text was added.
### How has it been tested?
I tried some time before and after the "FROM" time in the "TO" time picker.

### CODE REVIEW

File: forms/offer/transport/OfferTransportGoodsForm.tsx [hunk 1]
```diff
@@ -58,7 +58,11 @@ export const OfferTransportGoodsForm = ({
       .required(t('error.availability.required'))
       .typeError(t('error.must.be.string')),
     availability_interval_from: yup.mixed().typeError(t('error.must.be.time')),
-    availability_interval_to: yup.mixed().typeError(t('error.must.be.time')),
+    availability_interval_to: yup.mixed().typeError(t('error.must.be.time'))
```
Comment (idormenco) [inline forms/offer/transport/OfferTransportGoodsForm.tsx:61]: 1. this validation should be added in `OfferTransportPersonsForm.tsx` also
2. please extract this validator in `validators.ts` see `dateStringValidator` or `dateInTheFutureValidator`

File: public/locales/ru/common.json [hunk 2]
```diff
@@ -119,6 +119,7 @@
   "services.county_coverage": "Округ",
   "error.must.be.string": "Значение должно содержать символы",
   "error.must.be.time": "Значение должно быть час:минута",
+  "error.must.be.greater.than.from": "Until value must be greater than From",
```
Comment (idormenco) [inline public/locales/ru/common.json:122]: you can remove this line and we will provide translation

File: public/locales/ua/common.json [hunk 3]
```diff
@@ -118,6 +118,7 @@
   "services.county_coverage": "Округ",
   "error.must.be.string": "Значення має містити символи",
   "error.must.be.time": "Значення має бути година: хвилина",
+  "error.must.be.greater.than.from": "Until value must be greater than From",
```
Comment (idormenco) [inline public/locales/ua/common.json:121]: you can remove this line and we will provide translation

**Review by idormenco — CHANGES_REQUESTED** [review]

**Review by cipick — APPROVED** [review]
LGTM

**Review by idormenco — APPROVED** [review]

**Pull request merged**